tools/build_api.py

```diff
@@ -190,16 +190,23 @@ def build_project(src_path, build_path, target, toolchain_name,
 
     try:
         # Scan src_path and libraries_paths for resources
-        resources = toolchain.scan_resources(src_paths[0])
-        for path in src_paths[1:]:
-            resources.add(toolchain.scan_resources(path))
+        resources = sum(map(toolchain.scan_resources,src_paths), None)
```

Comment:
The old implementation is cleaner to read and makes it obvious that `resources` is not a list.